### ISSUE TITLE
Arrows previous and next are not visible upon receiving keyboard focus

### ISSUE THREAD

**Issue by hannolans** [UniversalViewer/universalviewer#461]
Arrows previous and next are not visible upon receiving keyboard focus. Visibility is triggered by mouse only.

**Comment by hannolans** [comment]
This is still the case. Tab focus on the rotate button:
![schermafbeelding 2018-02-09 om 16 44 51](https://user-images.githubusercontent.com/899399/36035909-a74cd8f6-0db8-11e8-9536-67a5e527abf3.jpg)
Then one tab further, focus on the 'previous arrow':
![schermafbeelding 2018-02-09 om 16 44 58](https://user-images.githubusercontent.com/899399/36035933-b8775c28-0db8-11e8-9be0-0b515fd5c6db.jpg)

**Comment by mejackreed** [comment]
I think this may be resolved in current `master` (if I understand the issue correctly).

See current tab focus changes (Chrome)

![uvfocus](https://user-images.githubusercontent.com/1656824/36036597-813082c6-0d77-11e8-8879-06b5c481952c.gif)
